tests/e2e/specs/backend/product-results-count.test.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import {
+	canvas,
+	createNewPost,
+	insertBlock,
+	switchUserToAdmin,
+} from '@wordpress/e2e-test-utils';
+import { searchForBlock } from '@wordpress/e2e-test-utils/build/inserter';
+
+/**
+ * Internal dependencies
+ */
+import {
+	filterCurrentBlocks,
+	goToSiteEditor,
+	useTheme,
+	waitForCanvas,
+} from '../../utils.js';
+
+const block = {
+	name: 'Product Results Count',
+	slug: 'woocommerce/product-results-count',
+	class: '.wc-block-product-results-count',
+};
+
+describe( `${ block.name } Block`, () => {
+	describe( 'in a block page', () => {
```

Comment:
Nit: can we make the same changes from https://github.com/woocommerce/woocommerce-blocks/pull/8157/commits/895f90e38b3268cb88366a081ca68cdd2d812203?